src.py

```
@@ -0,0 +1,54 @@
+import subprocess
+import json
+import os
+from datetime import datetime
+import pandas as pd
+
+
+def az_storage_blob_list(
+    prefix, 
+    cost_management_key, 
+    cost_management_storage_container, 
+    cost_management_storage_account
+):    
+    script_path = f"{os.path.dirname(__file__)}/az-storage-blob-list.sh"
+    return run_bash_get_json([
+        script_path,
+        prefix,
+        cost_management_key, 
+        cost_management_storage_container, 
+        cost_management_storage_account])
+
+
+def run_bash_get_json(args):
+    process = subprocess.run(args, capture_output=True, text=True)
+
+    if process.returncode == 0:
+        json_data = json.loads(process.stdout)
+    else:
+        # handle this better...
+        print('something went wrong')
+        json_data = None
```

probably better to raise an exception than to return None